src/main/scala/io/github/sentenza/hacktoberfest18/sort/CommonSortings.scala

```diff
@@ -1,5 +1,7 @@
 package io.github.sentenza.hacktoberfest18.sort
 
+import java.util.Random
+
```

Comment:
We don't need this import statement any more, do we?

Reply:
No.